buku

```diff
@@ -3612,7 +3569,7 @@ def get_firefox_profile_names(path):
     from configparser import ConfigParser, NoOptionError
 
     profiles = []
-    profile_path = os.path.join(path, 'profiles.ini')
+    profile_path = os.path.expanduser(os.path.join(path, 'profiles.ini'))
```

Comment:
Since the expansion is necessary either way, it's more reliable to just do it within the function, instead of relying on it be done by the invoking code.